inspirehep/modules/workflows/workflows/article.py

```diff
@@ -199,6 +200,7 @@
     guess_keywords,
     guess_coreness,
     normalize_collaborations,
+    normalize_affiliations
```

Comment:
This should only happen for core records, so it can't happen here, you need to do it in post-enhance (so you know if a record is core) and check whether the record is core.

Reply:
Ah ok, thank you!